index.html

```diff
@@ -6,14 +6,1394 @@
+                    </li>
+                    <li class="drop-down">
+                        <a href="#">Bussiness <i class="fas fa-caret-down"></i></a>
+                        <ul class="dropdown-content reduce-size">
+                            <li><a href="#">Index</a></li>
+                            <li><a href="#">TNW X</a></li>
+                        </ul>
+                    </li>
+                    <li><a href="#">AMAs</a></li>
+                    <li><a href="#">Spaces</a></li>
+                    <li><a href="#">Tearms and Conditions</a></li>
+                </ul>
+                <ul class="pop-out-right">
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Advertise</a></li>
+                    <li><a href="#">Jobs</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+            <div class="nav-bottom-static">
+                <ul class="nav-section">
+                    <li><a href="#" class="first">Latest</a></li>
+                    <li><a href="#">Hard fork</a></li>
+                    <li><a href="#">Plugged</a></li>
+                    <li><a href="#">Fundamentals</a></li>
+                    <li><a href="#">REadme</a></li>
+                    <li class="display-none"><a href="#">Growth Quarters</a></li>
+                    <li class="display-none"><a href="#">Neural</a></li>
+                </ul>
+                <ul class="nav-icons">
+                    <li>
+                        <a href="#" class="search"> <i class="fas fa-search"></i></a>
+                    </li>
+                    <li class="mob-display">
+                        <a href="#"> <i class="fas fa-bars red-light"></i></a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <!-- Main body -->
+    <main class="main">
+        <section class="cover cover-grid-3">
+            <article class="cover-story big-cover">
+
+                <div class="cover-image lazy lazyLoaded" sizes="(max-width: 796px) 100vw, 796px" width="796"
+                    height="417" srcset="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
+                    style="background-image: url(&quot;https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/04/Animal-Crossing-New-Horizons-796x417.jpg&quot;);">
+                </div>
+                <a class="cover-trigger"
+                    href="https://thenextweb.com/gaming/2020/05/08/detroit-lions-use-animal-crossing-reveal-2020-schedule/"
+                    data-event-category="Homepage" data-event-action="Top story"
+                    data-event-label="Detroit Lions use Animal Crossing (yes, really) to reveal 2020 schedule"
+                    data-event-non-interaction="true"></a>
+                <div class="cover-gradient">
+
+                </div>
+                <div class="cover-logo"></div>
```

Comment:
The logo crops when I am reducing the size of the page. Kindly fix that.

Screenshot from your page. 

![image](https://user-images.githubusercontent.com/57414789/81482979-69bef900-9200-11ea-9e9e-9c0618534977.png)